services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationDataMapper.kt

```diff
@@ -15,38 +18,92 @@ object ApplicationDataMapper {
         applicationData: CableReportApplicationData
     ): AlluCableReportApplicationData =
         with(applicationData) {
-            val description = workDescription(cableReport = this)
+            val description = workDescription(workDescription, rockExcavation!!)
             AlluCableReportApplicationData(
+                postalAddress = postalAddress?.toAlluData(),
                 name = name,
                 customerWithContacts =
                     customerWithContacts
                         .orThrow(path("customerWithContacts"))
                         .toAlluData(path("customerWithContacts")),
-                geometry = getGeometry(applicationData = this),
+                representativeWithContacts =
+                    representativeWithContacts?.toAlluData(path("representativeWithContacts")),
+                invoicingCustomer = null,
+                geometry = getGeometry(this),
                 startTime = startTime.orThrow(path("startTime")),
                 endTime = endTime.orThrow(path("endTime")),
                 pendingOnClient = pendingOnClient,
                 identificationNumber = hankeTunnus,
+                customerReference = null,
+                area = null,
```

Comment:
We need to send the area to Allu. Currently, the frontend calculates the area and the backend trusts the frontend, which is far from ideal. But I don't think it's something that should be fixed in this PR.

Reply:
Or so I thought. Looks like the frontend sends it as null.

If it's always null, it's not included in the JSON anyway, so I'm not sure how useful it is in the class.